web/src/main/java/pl/edu/agh/rest/login/entity/LoginRequestDto.java

```diff
@@ -0,0 +1,19 @@
+package pl.edu.agh.rest.login.entity;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableLoginRequestDto.class)
+@JsonDeserialize(as = ImmutableLoginRequestDto.class)
+public interface LoginRequestDto {
```

Comment:
`LoginRequestDTO`

Reply:
Uuu, to jest kontrowersyjny temat. Ja się chyba trzymam konwencji, gdzie się nie pisze skrótów całych wielkimi literami tylko właśnie `Dto`

[Google](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) też tak w sumie mówi, bo potem jak się pojawiają cuda typu `XMLHTTPRequest`, to to lepiej wygląda

Ale możemy się o to jeszcze pokłócić jak najbardziej, bo pewnie są też w internecie miejsca, które mówią odwrotnie

Reply:
Obawiam się, że i tak nie będziemy mieć wpływu na to, co nam swagger wygeneruje

Reply:
No na to nie (swagger zostawia puste linijki przed `}` na końcu!!1), ale jak coś piszemy sami, to generalnie mamy. Co jeśli spotkamy na swojej drodze więcej akronimów?